src/main/java/nextstep/subway/favorite/domain/Favorite.java

```diff
@@ -0,0 +1,58 @@
+package nextstep.subway.favorite.domain;
+
+import nextstep.subway.BaseEntity;
+import nextstep.subway.common.exception.InvalidDataException;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.util.List;
+
+@Entity
+public class Favorite extends BaseEntity {
+
+    private static final String IDENTICAL_STATION_EXCEPTION = "출발역과 도착역이 동일합니다.";
+    private static final String FAVORITE_ALREADY_EXITS_EXCEPTION = "해당 구간은 사전에 등록되었습니다.";
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Long sourceId;
+    private Long targetId;
+
+    public Favorite() {
```

Comment:
```suggestion
    protected Favorite() {
```

JPA 의 기술 구현 상 빈 생성자를 없애는건 불가능합니다만,
접근 가능 범위를 최소한으로 줄여주는건 어떤가요?

Reply:
넵 수정했습니다